src/tools/rest-backend/cryptlite/base64.h

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
CppCMS is using a special base64 encoding adapted for use in URLs (some URL characters like `/` are not used). The CppCMS base64 class would probably be able to replace the cryptlite base64 functions, but I would still have to keep the cryptlite/base64.h for the cryptlite/sha256.h header. So I'm not sure if it is worth the effort.

As the cryptlite headers are a header-only interface for the boost crypt library anyway, I'm not sure if it is a viable option to try to replace it by Boost functions directly. It seems to me as if this would result in less readable code with exactly the same dependencies...